lib/controllers/DeviceController.ts

```diff
@@ -248,6 +252,24 @@ export class DeviceController extends CRUDController {
       { query: { bool: { filter } } });
   }
 
+  async importDevices (request: KuzzleRequest) {
+    const body = request.getBody();
```

Comment:
Instead of retrieving the body and manually check for the `csv` string property presence, you can directly use the `getBodyString` method. In that way, a standard error message will be returned if the string does not exists